### DISCUSSION TITLE
Wayland (labwc), multiple monitors, and keeping Conky on a specifc one

### DISCUSSION THREAD

**Discussion by MrArborsexual** [brndnmtthws/conky#1628] [Q&A]
Been a VERY long time since I last used conky, and I'm a very recent Wayland convert. I'm using labwc because I am coming from openbox, and it is familiar. My conky config isn't even a config yet, just a scrolling banner because I want to get the orientation right and play with some colors before I really dig into making it a full config.

I currently have two displays (eDP-1, and DP-2). eDP1 is the screen for my laptop, and DP-2 is the monitor I actually do things on. I want eDP-1 to basically be a pretty picture with a dedicated conky overlay. I tried specifying the display in my config, but when I run 'conky' it starts on whichever screen the mouse is on, and even if I make sure it starts on eDP-1, if I change the config and save, conky then moves back to DP-2.

Is there a way in wayland to specify where conky is located?

I was thinking maybe specifying coordinates (per wlr-randr, eDP-1 is located at 0,0 and DP-2 is located at 1280,0), but don't see an obvious way to do that (maybe there is though?). I want to figure this out now, as I will be adding two more monitors (really old TVs for retro gaming) which will make for an even more complicated monitor arrangement.

Config:
conky.config = {
	alignment = 'top_left',
	background = true,
	border_inner_margin = 2,
	border_outer_margin = 2,
	border_width = 1,
	cpu_avg_samples = 2,
	default_color = 'white',
	display = 'eDP-1',
	double_buffer = true,
	font = 'Fira Code:size=12',
	forced_redraw = true,
	format_human_readable = true,
	net_avg_samples = 2,
	out_to_x = false,
	out_to_wayland = true,
	own_window = true,
	own_window_argb_visual = true,
	own_window_argb_value = 57,
	own_window_color = blue,
	own_window_class = 'Conky',
	own_window_type = 'desktop',
	update_interval = 1.0,
	use_xft = true,
}
	
conky.text =[[
	${color #4a148c}Info: ${scroll 32 Conky $conky_version - $sysname $nodename $kernel $machine}
]]

**Comment by thefrog-Argedion** [comment]
take this
alignment = 'top_left', and change it to alignment = 'none',
gap_x=$VALUE,
gap_y=$VALUE,
